client/my-sites/stats/stats-email-chart-tabs/utility.js

```diff
@@ -57,10 +57,13 @@ export const buildChartData = memoizeLast( ( activeLegend, chartTab, data, perio
 function addTooltipData( chartTab, item, period ) {
 	const tooltipData = [];
 	const label = ( () => {
+		const formatedDate = formatDate( item.data.period, period );
```

Comment:
really small typo here, `formatedDate` should be `formattedDate` (note the `tt`)